doc/mediasdk-man.md

```diff
@@ -7564,6 +7570,37 @@ This structure is used to pass decryption status report index for Common Encrypt
 
 This structure is available since SDK API 1.30.
 
+## <a id='mfxExtPartialBitstreamParam'>mfxExtPartialBitstreamParam</a>
+
+**Definition**
+
+```C
+typedef struct {
+    mfxExtBuffer    Header;
+    mfxU32          BlockSize;        /* output block granulatiry for Granularity = MFX_PARTIAL_BITSTREAM_BLOCK */
+    mfxU16          Granularity;      /* granulatiry of the partial bitstream: slice/block/any */
+    mfxU16          reserved[8];
+} mfxExtPartialBitstreamParam;
+```
+
+**Description**
+
+This structure is used by AVC HW encoder to output parts of bitstream as soon as they ready. The application can attach this extended buffer to the [mfxVideoParam](#mfxVideoParam) structure at init time.
+If this option is turned ON (Granularity != MFX_PARTIAL_BITSTREAM_NONE) and session configured in single thread (session initialized with MFX_IMPL_EXTERNAL_THREADING), then AVC encoder can output bitstream by part based with required granularity. <br>
```

Comment:
You refer to something which is not described in the documentation, but exists in the header file and in mediasdk dispatcher:
```
[dvrogozh@dvrscl msdk]$ fgrep -rsn MFX_IMPL_EXTERNAL_THREADING
api/include/mfxcommon.h:67:    MFX_IMPL_EXTERNAL_THREADING        = 0x10000,
api/mfx_dispatch/windows/src/main.cpp:68:        {MFX_LIB_HARDWARE, MFX_IMPL_HARDWARE|MFX_IMPL_EXTERNAL_THREADING, 0},
api/mfx_dispatch/windows/src/main.cpp:69:        {MFX_LIB_HARDWARE, MFX_IMPL_HARDWARE2|MFX_IMPL_EXTERNAL_THREADING, 1},
api/mfx_dispatch/windows/src/main.cpp:70:        {MFX_LIB_HARDWARE, MFX_IMPL_HARDWARE3|MFX_IMPL_EXTERNAL_THREADING, 2},
api/mfx_dispatch/windows/src/main.cpp:71:        {MFX_LIB_HARDWARE, MFX_IMPL_HARDWARE4|MFX_IMPL_EXTERNAL_THREADING, 3},
api/mfx_dispatch/windows/src/mfx_library_iterator.cpp:66:    mfxIMPL impl_via = (*pImplInterface & ~MFX_IMPL_EXTERNAL_THREADING);
```

This gives a bunch of questions what is this mode, is it really functional and is it really implemented?